public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -1967,6 +1978,7 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
     self.timeoutWalking = timeoutWalking;
     self.resetWalking = resetWalking;
     self.getMoveDelay = getMoveDelay;
+    // this.makeLinksClickable = makeLinksClickable;
```

Comment:
You can just delete this line.

Reply:
Okay I think that this is the only thing that you haven't resolved. There are 3 lines of commented out code that you've added. Two in `SVLabel/Main.js` and one in `MapService.js`.